darwin/client.py

```diff
@@ -237,7 +237,8 @@ def delete(
         return self._decode_response(response, debug)
 
     def list_local_datasets(self, team: Optional[str] = None) -> Iterator[Path]:
-        """Returns a list of all local folders which are detected as dataset.
+        """
+        Returns a list of all local folders which are detected as dataset.
```

Comment:
A folder is considered a dataset folder if the has a subfolder called "releases" and a subfolder called "images".
You can double check my interpretation of the code by checking out `utils.py` line 43.

```
def is_project_dir(project_path: Path) -> bool:
    """
    Verifies if the directory is a project from Darwin by inspecting its structure.

    Parameters
    ----------
    project_path : Path
        Directory to examine

    Returns
    -------
    bool
        Is the directory a project from Darwin?
    """
    return (project_path / "releases").exists() and (project_path / "images").exists()
```

I chose not to document what a dataset folder is, as this is business logic which can change. When that happens, the documentation will not only be obsolete, it will also be incorrect. Because this piece of documentation is not in the function itself, I find it better (if someone is interested) to check the code itself as I find it clear enough (no indirection :P)